CMakeLists.txt

```diff
@@ -129,10 +129,11 @@ if(USE_SSL)
       message(FATAL_ERROR "Neither OpenSSL nor WolfSSL was found; please install a devel package")
     endif()
   endif()
+  add_definitions("-DUSE_TLS" "-DUSE_SHA256")
```

Comment:
This is bad. Please change line 140 to `if(USE_SSL)`, and move this section there (the existing block should be in `else(USE_SSL)`).

Reply:
See https://github.com/orgads/sipp/commit/f30eeb50127ea8b876cc8819dc607959d2b88a40

Reply:
@orgads please take a look https://github.com/SIPp/sipp/pull/676/commits/42eac7f0a0c8c65e8d183e771010e70665d68429.

Reply:
Still not good. `USE_SSL` can be changed to 0 in this scope.

Reply:
Oh, I see. I didn't notice the recent change in master branch
```
message(STATUS "Neither OpenSSL nor WolfSSL was found; disabling sips support")
set(USE_SSL 0)
```
Fixed.